src/Core/Components/NavMenu/FluentNavGroup.razor.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Web;
+using Microsoft.Fast.Components.FluentUI.Utilities;
+
+namespace Microsoft.Fast.Components.FluentUI;
+
+public partial class FluentNavGroup : FluentNavBase
+{
+    private readonly RenderFragment _renderContent;
+    private readonly RenderFragment _renderButton;
+
+    protected string? ClassValue =>
+        new CssBuilder("fluent-nav-group")
+            .AddClass("fluent-nav-item")
+            .AddClass("expanded", Expanded)
+            .AddClass("disabled", Disabled)
+            .AddClass(Class)
+            .Build();
+
+    internal string? StyleValue => new StyleBuilder(Style)
+       .AddStyle("margin", $"{Gap} 0" , !string.IsNullOrEmpty(Gap))
```

Comment:
I'm not sure, but I think you have to set the unit after zero: `0px`

Reply:
> Specifying the length unit is optional if the number is 0.

[MDN length](https://developer.mozilla.org/en-US/docs/Web/CSS/length)

Some of the CSS editors actually complain if you _do_ add a unit to 0. I actually thought it wasn't supposed to be there because of that, not just optional.

Reply:
In Dev Tools it show as `0` when combined and as `0px` when separated out:
<img width="141" alt="image" src="https://github.com/microsoft/fluentui-blazor/assets/1761079/b51e1910-0934-4c0b-8ca1-6e35e1f381a5">
I'll leave it as-is.